packages/backend/utils/sign.js

```diff
@@ -124,6 +124,15 @@ const getSignMessageForId = async (messageId, options) => {
         reachedOut ? "reached out" : "NOT reached out"
       }`;
 
+    case "builderUpdateBatch":
+      // We do this because of the way GET params work (can't send JS objects as in POST requests)
+      if (typeof options.batch === "string") {
+        data = options.batch;
+      } else {
+        data = JSON.stringify(options.batch);
+      }
+      return `I want to set my batch data as ${options.address}:\n\n${data}`;
```

Comment:
This could be a bit more friendly:

"I want to request telegram access for BATCH X as {address}"

Reply:
Aggree, will change it :)

Reply:
![Screenshot 2024-10-02 at 17 26 24](https://github.com/user-attachments/assets/a792345b-2691-43de-bb56-ba656ad738d5)